### PR TITLE
Side nav changes

### PR DESCRIPTION
**Pull request by h-kataria**
- changed urls from ansible/insights/ to ansible/automation-analytics

https://issues.redhat.com/browse/AA-1114

### CODE REVIEW

**Review by Hyperkid123 — DISMISSED** [review]
The `automationAnalytics` module does not have route registered for `/ansible/automation-analytics`

Can you please update the `fed-modules.json` file?

**Review by karelhala — COMMENTED** [review]
You should also update `automationAnalytics` federated module in `fed-mods.json` with `/ansible/automation-analytics` route. Otherwise this will throw an error of unknown application. Can we also test this on `prod-beta` first? A different PR to prod-beta would be required so we can test if everything works and perhaps could roll back if it would throw some errors.

File: chrome/fed-modules.json [hunk 1]
```diff
@@ -379,7 +379,7 @@
                 "id": "ansible-automation-analytics",
                 "module": "./RootApp",
                 "routes": [
-                    "/ansible/insights",
+                    "/ansible/automation-analytics",
                     "/ansible/reports",
```
Comment (Hyperkid123) [inline chrome/fed-modules.json:384]: What about the rest of the routes on the list? Is the automation analytics app supposed to render there? I would guess no based on the links from the nav.

Comment (h-kataria) [reply chrome/fed-modules.json:384]: @Hyperkid123 not sure how are they working currently in that case, I was confused hence did not change rest of them

Comment (Hyperkid123) [reply chrome/fed-modules.json:384]: OK lest take clean them up in another PR

**Review by karelhala — APPROVED** [review]

**Pull request merged**